Safewaan/HonkedIn
# Add Jest and Cypress dependencies for testing

## Implementation description
* Added dependencies for Jest testing and Cypress e2e testing
* Created configuration files for tests
* Created sample test file for Jest and Cypress



## Steps to test
1. Verify that the Jest test passes. Run `npm run test`.
2. Verify Cypress is configured and the test runs. Run `npm run cypress` and ensure the application is running. Click on the smoke test and verify it passes.



## What should reviewers focus on?
*
